Source/Core/Common/GL/GLInterface/WGL.cpp

```diff
@@ -400,12 +459,14 @@ HGLRC GLContextWGL::CreateCoreContext(HDC dc, HGLRC share_context)
     HGLRC core_context = wglCreateContextAttribsARB(dc, share_context, attribs.data());
     if (core_context)
     {
-      INFO_LOG_FMT(VIDEO, "WGL: Created a GL {}.{} core context", version.first, version.second);
+      INFO_LOG_FMT(VIDEO, "WGL: Created a GL{} {}.{} core context",
+                   *mode == Mode::OpenGLES ? "ES" : "", version.first, version.second);
       return core_context;
     }
```

Comment:
As a point for discussion: does it make sense to refer to a "GLES 3.2 core context", or is a core context only a meaningful concept on desktop OGL?

From what I can tell, the `core` parameter to `GLContext::Create` is always set to true (both in [OGLMain](https://github.com/dolphin-emu/dolphin/blob/20ac2cf781b1c514b4eea47e5da4195d95b5bad8/Source/Core/VideoBackends/OGL/OGLMain.cpp#L175-L179) and [SWOGLWindow](https://github.com/dolphin-emu/dolphin/blob/20ac2cf781b1c514b4eea47e5da4195d95b5bad8/Source/Core/VideoBackends/Software/SWOGLWindow.cpp#L35-L39) (using [the default arguments](https://github.com/dolphin-emu/dolphin/blob/20ac2cf781b1c514b4eea47e5da4195d95b5bad8/Source/Core/Common/GL/GLContext.h#L47-L51))), which also seems weird.